### ISSUE TITLE
Upload to Pip

### ISSUE THREAD

**Issue by iTaybb** [tjguk/active_directory#3]
Hi,
I really like your module. But sadly it isn't available in Pypi.
Can you please upload it?

Thanks!

**Comment by woodlandhunter** [comment]
@iTaybb Are you the person who registered active_directory on pypi ? Would you be willing to coordinate with me to get an updated version that also works under py3 uploaded?

**Comment by iTaybb** [comment]
Yeah I would like to. I actually wanted to make a py3 version myself but
didn't find the time to do it.
